### DISCUSSION TITLE
Including precise touch patch into linux-surface

### DISCUSSION THREAD

**Discussion by Freek07** [linux-surface/linux-surface#970] [General]
Hi there,

as note 14 in the feature matrix suggests, pen input works on SP7 after patching the driver using (https://github.com/linux-surface/intel-precise-touch), recompiling it and using latest iptsd from the github.

Are there any plans to at least include the kernel driver patch in the linux-surface package? It would make it much easier to keep the system updating after kernel upgrades :-)

Thanks,

May

**Comment by qzed** [comment]
It will be included when it's ready, i.e., when it's reasonably stable and documented. @StollD has been working on that. I think the target time-frame was end of this year.
